repos/system_upgrade/el7toel8/models/systemfacts.py

```diff
@@ -23,7 +23,8 @@ class RepositoryData(Model):
     topic = SystemInfoTopic
 
     name = fields.String()
-    baseurl = fields.String()
+    baseurl = fields.Nullable(fields.String())
+    metalink = fields.Nullable(fields.String())
```

Comment:
I see here problem, that only one of them (`baseurl` or `metalink`) can be set and the other one has to be `None`. In addition, there can be even `mirrorlist` regarding the [documentation](https://dnf.readthedocs.io/en/latest/conf_ref.html#mirrorlist-label). So at least this one is still missing when we are fixing the model. I am going to test behaviour when more then one of the parameters are set.